inheritance/src/main/java/inheritance/Library.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package inheritance;
+
+public class Library {
```

Comment:
Feel free to delete this file (Library and LibraryTest) once your project is up and running